modules/publicize/assets/publicize.css

```diff
@@ -49,11 +49,6 @@ div#tumblr:before {
 	content: '\f214';
 }
 
-div#google_plus:before {
```

Comment:
I don't recall -- do we need to manually make the RTL/MIN versions of this CSS or is it part of the building process?

Reply:
Hm… I thought it was part of the build process, but I'm not sure.

Reply:
It looks like RTL is still manually committed after running through dotcom's autortl script. We should get this moved over to automated things, but that's out of scope for this PR.

Worst case, if it gets left in this release, not the end of the world.

Reply:
I don't think it matters :) Neither the RTL nor the LTR files are ever loaded anywhere now.

Reply:
File removal in #11426.